docs/src/app/navigationConstants.js

```diff
@@ -297,6 +297,16 @@ export const navigationItems = [
                         label: 'Tabs',
                         path: 'tabs',
                     },
+                    {
+                        component: 'navigationTabs',
```

Comment:
I am thinking we should honestly rename this to maybe "Tab Navigation." I am discussing right now, but my quick thoughts are we have two different types of navigation here: Sidebar Navigation and Tab Navigation. Also, another thought is we cannot call it simply Tabs, which honestly is probably good, because look at the situation we are already in by being to simple.

Reply:
Maybe _Tabbed_ Navigation ...?

Reply:
Done